src/wasm-type.h

```diff
@@ -24,7 +24,11 @@
 namespace wasm {
 
 class Type {
-  uint32_t id;
+  // enough for the limit of 1000 function arguments
+  static constexpr unsigned sizeBits = 10;
```

Comment:
perhaps `SizeBits` (capitalize?) or `SIZE_BITS`? we seem to use both of those, and to leave `lowerCaps` style names for actual variables.